plugins/@grouparoo/postgres/src/lib/export/exportRecord.ts

```diff
@@ -29,14 +27,86 @@ export const exportRecord: ExportRecordPluginMethod = async ({
     );
   }
 
-  try {
-    // --- Records --- //
-    if (toDelete) {
-      if (!syncOperations.delete) {
+  // --- Records --- //
+  if (toDelete) {
```

Comment:
The linter noticed this: https://eslint.org/docs/rules/no-unsafe-finally
Since we were throwing the error if it happened we didn't need a try/catch.
Functionality doesn't seem to have changed but I imagine this fixes a weird edge case.